BUILDING_FROM_SOURCE.md

```diff
@@ -104,25 +104,33 @@ For any OS, you can use [`zigup`](https://github.com/marler8997/zigup) to manage
 If you prefer a package manager, you can try the following:
 
 - For MacOS, you can install with `brew install zig@0.11.0`
-- For, Ubuntu, you can use Snap, you can install with `snap install zig --classic --beta`
+- For systems with Snap, you can install with `snap install zig --classic --beta`
```

Comment:
I think referencing Ubuntu by name here is probably helpful to people running Ubuntu.

How about this?

```suggestion
- For systems with Snap (such as Ubuntu), you can install with `snap install zig --classic --beta`
```